src/LevelAgreement.php

```diff
@@ -1013,8 +1013,33 @@ public function addLevelToDo(Ticket $ticket, $levels_id = 0)
 
         if ($levels_id) {
             $toadd = [];
+
+            // Compute start date
+            if ($pre == "ola") {
+                // OLA have their own start date which is set when the OLA is added to the ticket
+                if (
+                    $this->fields['type'] == SLM::TTO
```

Comment:
Special dedication to PHP 7.4
```suggestion
                    (int)$this->fields['type'] === SLM::TTO
```